agpypeline/entrypoint.py

```diff
@@ -368,7 +373,8 @@ def add_parameters(parser: argparse.ArgumentParser, algorithm_instance: Algorith
         algorithm_instance.add_parameters(parser)
 
     # Assume the rest of the arguments are the files
-    parser.add_argument('file_list', nargs=argparse.REMAINDER, help='additional files for transformer')
+    parser.add_argument('file_list', nargs='+', type=str,
+                        help='additional files for transformer')
```

Comment:
Why was this change needed?

Reply:
I based off of the comments from https://github.com/kyclark/transformer-canopycover/blob/kyc_testing/REVIEW.adoc . And then noted (I thin it might have been in an earlier message from slack) that argparse.FileType opens the file and leaves it open, which can cause issues and so changed it to type=str since otherwise a chunk of the agpypeline code would need to be changed. (Let me know if this should be done instead though)

<img width="1168" alt="Screen Shot 2021-01-08 at 11 33 53 AM" src="https://user-images.githubusercontent.com/62153719/104051463-6459ea00-51a5-11eb-80f9-203f5b71da12.png">
<img width="1020" alt="Screen Shot 2021-01-08 at 11 34 14 AM" src="https://user-images.githubusercontent.com/62153719/104051490-70de4280-51a5-11eb-9358-17f4f9f9a593.png">

Reply:
That argument is supposed to be optional.